index.html

```diff
@@ -9,17 +9,31 @@
 <body>
     <main>
         <h1>Amazing Library App</h1>
-        <div id="books-container">
-            <div class="book-title">
-
+        <form id="add-book">
+            <div class="form-data">
+                <label for="title">Title</label>
+                <input type="text" id="title">
             </div>
-            <div class="author">
-
+            <div class="form-data">
+                <label for="author">Author</label>
+                <input type="text" id="author">
             </div>
-            <div class="pages-number">
-
+            <div class="form-data">
+                <label for="pages">No.Pages</label>
+                <input type="number" id="pages">
             </div>
-        </div>
+            <input type="submit" value="Add a Book" id="button">
+        </form>
```

Comment:
- [ ] [REQUIRED] This form should only be visible after the user clicks on "new book", and it should also include a field to check if the user already read the book or not BEFORE adding the book to the library.
![image](https://user-images.githubusercontent.com/73254040/99810823-e7e24d80-2b09-11eb-9bed-c996bbf36170.png)
[Source](https://www.theodinproject.com/courses/javascript/lessons/library)